### ISSUE TITLE
Please Remove/Delist GMT 

### ISSUE THREAD

**Issue by ryanozo** [forkdelta/tokenbase#906]
Contract address: 0xb3bd49e28f8f832b8d1e246106991e546c323502

Issuer's official website: mercuryprotocol.com

Official contract address confirmation:
 0x21f743ec5a2271fb0f4c83eea77b0986ed069c101daa2ab90fcffca17f94911f4d0437140ed5b1fa824472d582ab6149c681d80030611739148a45ab8c5339e51c

A link to a third-party review or discussion of the token:

 

http://fortune.com/2017/08/24/mark-cuban-blockchain-mercury-protocol/

Description: Dear ForkDelta, please delist Mercury Protocol's GMT utility token. Trading of GMT is not allowed by the original terms of sale. Thank you, Ryan Ozonian.

**Comment by freeatnet** [comment]
Hey Ryan,

Could you please include the link to the document prohibiting trading of GMT and the quote the relevant sections of it directly?

Signature checks out:
```
Dear ForkDelta, please delist Mercury Protocol's GMT utility token. Trading of GMT is not allowed by the original terms of sale. Thank you, Ryan Ozonian.
```

signed with 0x85c762c235b2fd8df2038e3bd4691f295ea2fe91 ([GMT.owner](https://etherscan.io/address/0xb3bd49e28f8f832b8d1e246106991e546c323502#readContract)), see https://etherscan.io/verifySig/255

**Comment by ryanozo** [comment]
Per our Terms of Sale item 10 (p), trading GMT is forbidden. Terms of Sale may be found here: https://www.mercuryprotocol.com/files/Radical_App_International_Terms_of_Token_Sale.pdf

Best,
Ryan

**Comment by freeatnet** [comment]
Thanks @ryanozo.

Next steps here:
1. Announce this request to the community (today, April 11th). Announcements will be posted on our Reddit, Twitter, Telegram, and Discord. They should be cross-posted to GMT social channels.
2. Allow for comments period of 1 week, allow users to withdraw their funds in this timeframe.
3. Assuming no significant concerns about validity of this request, I will remove GMT listing on April 18th.

**Comment by freeatnet** [comment]
Dear community:
We have received this request to remove the listing for Global Messaging Token (GMT) by mercuryprotocol.com. To the best of our knowledge, this request is authentic. Barring significant concerns about legitimacy of this request, we will proceed by removing the listing on April 18th, 2018.

We would like to allow everyone an opportunity to review the request and voice any concerns.

**Comment by rartokens** [comment]
Hi @ryanozo ,
are you connected to the company somehow to know this thing?

**Comment by Arctek** [comment]
I would like to raise a concern.
Many of us have traded this token post-ico, never agreed to the terms and as such are not bound by them.
Additionally the token is an unlocked ERC20 contract, which implicitly means it can be traded without any restrictions, legal or technical.
If the original intention had been for this to be non-tradeable then it should have never been an ERC20, or had a lockup associated with it.

Until this concern is adequately addressed for those who now have GMT balances, I would humbly request that the listing remain.

**Comment by shalabolic** [comment]
I agree with Arctek. If they did not want the token to be traded they should not have used the ERC20 platform or had the token locked. They made their bed so now can lay in it..

**Comment by J-Dream** [comment]
Also,  I think Arctek's concern is incredibly valid.   Most GMT owners did not sign or agree not to sell the tokens.   In fact we were led to invest because we believed we would be able to freely trade our assets based on the fact that it is an unlocked ERC20 contract and was listed on an exchange.   Removing our rights to trade our personal assets does not seem legitimate or reasonable, no matter who made the request.   Digital assets are people's personal property.   As an ERC20 token we believed that no single entity could prevent us from exchanging our assets between individuals.   And so it seems like a violation property rights to forcibly prevent us from doing so and thereby force us to eat major financial losses.

It would be greatly appreciated to hear updates if these concerns have any impact on the Forkdeta's decision to comply with the request.    (preferably before final day of trade.)

**Comment by samoskill** [comment]
If GMT is requesting delist, are they going to buy off all our tokens? I own several thousands, it would be great to get back my investment atleast please.

**Comment by freeatnet** [comment]
@ryanozo I think there may be some questions above you'd like to address.

**Comment by litsh27** [comment]
As an owner of several thousand GMT tokens, I second the question by @samoskill  put to @ryanozo. Please provide your community with some clear details how you intend to manage this situation, which at the moment comes across, at the very least, as somewhat suspicious.

**Comment by J-Dream** [comment]
@freeatnet since there seems to be legitimate concerns against delisting.   Can the date be moved back until adequate discussion has been heard.

**Comment by premorgue** [comment]
As long as you make and distribute an ERC20 token, you can not regulate trading it. It is solely the authority of the token owner. And you have left the trade in Etherdelta for over six months after ICO. Did not you explicitly deny the demand for listing on a number of exchanges and vaguely answer them? Why are you now coming to terms and trying to stop the deal?

**Comment by freeatnet** [comment]
@ryanozo Can you explain your position with respect to current token owners, specifically those who have purchased GMT tokens? Will their tokens be honoured?

**Comment by noelgit** [comment]
Something as important as the non-tradeability of this erc20 token should have been explicitly stated and explained in plain english before the ico and not 6 months later.

**Comment by ryanozo** [comment]
Token owners in good standing are a valuable part of our community, and we intend to honor our commitments to them.

**Comment by noelgit** [comment]
Delisting will effectively render the token worthless - no real use at the moment and no liquidity. Forkdelta will be complicit to this robbery if they allow it.

**Comment by JohnProfe** [comment]
This is a clear exit scam. They allowed the token to be traded and they probably sold all their holdings while it was going up. Now suddenly months later they want to remind the token holders on a contract they didnt sign up? Not only that but they clearly stated many times that they working on it about the listening on different exchanges and that nothing can be said about it. We didnt invest in this project for your sole benefit alone, i didnt buy into your ico nor im bound by any contract you made in a unragulated market. You either gona do what you have to do (buy back all our tokens) or we gona come and hunt you down with a class action law suite. This will not only frame your names on the whole internet but will also take a hefty damage on your company. You will not get away with this, i promise you.

**Comment by ryanozo** [comment]
I appreciate the comments and understand how this looks from the outside. Truthfully, we are simply trying to manage risk in the emerging regulatory environment.  Our team is diligently working in our West Los Angeles office (visitors welcome), we post regular updates on our blog, and like any other growing company we are evolving and adapting as we learn and test our products in the market.  As I said before, token holders in good standing are a valuable part of our community and we intend to fulfill promises made, and throughout our token sale we have presented clear terms of ownership. For those who purchased on secondary exchanges, I think you should direct your anger towards the sellers and exchanges, not at me.  At any time we could have been contacted by interested secondary market buyers to provide our original terms of sale (see my second comment above for the terms), and would have notified those potential buyers that we do not support secondary markets.  Thanks for the discussion and feedback, but we respectfully request again that GMT be removed.

**Comment by premorgue** [comment]
And do you think you will not support the GMT purchased at the secondary market at all? After the launch of "Real GMT", you seem to intend to make the purchased token from the secondary market into a piece of tissue. Ryan I want your answer to be clear.

**Comment by noelgit** [comment]
Believe this looks much much worse from the outside
- you got no working product. No, test net doesn't count
- you got no code pushes in github after the ico
- you removed community channels so you don't have to answer questions from the community
- now, you want to remove all forms of liquidity
This is beginning to look like a scam

**Comment by noelgit** [comment]
Believe me this looks much much worse from the outside
- you got no working product. No, test net doesn't count
- you got no code pushes in github after the ico
- you removed community channels so you don't have to answer questions from the community
- now, you want to remove all forms of liquidity

This is beginning to look like a scam

**Comment by J-Dream** [comment]
so whats the deal @freeatnet?     Are we just going to stay in limbo forever?  or you going to tell this clown that you support the rights of the token holders.   it seems like Ryanzo doesnt have a good leg to stand on and is making an unnessiary and reckless attempt to assert authority over who peoples etherium based assets and thereby cause major financial loss and upset to the community.     He pulls up some old contract for the ico that the vast majority of holders have never seen before or agreed to.    and asserts that the trading that has been happening for 6 months is no longer allowed.   

Also, item 10(p) of contract does NOT say trading is forbidden as @ryanozo claims.   
It says you must maintain control of your purchaser wallet etc.   I dont have a purchaser wallet and this item and contract have nothing to do with me.    And even more relevant to Forkdelta,  it certainly doesn't say that an exchange must prevent GMT holders from trading tokens.   

Here is what item 10p from the ICO "terms of sale" contract @ryanozo cited as validation for his request:
"You will at all times maintain control of your Purchaser Wallet, and you will not share or
disclose the account credentials associated with your Purchaser Wallet with any other
party. If you transfer GMT from your Purchaser Wallet into another wallet or vault, you
will likewise at all times maintain control of such other wallet or vault, and you will not
share or disclose the account credentials associated with such other wallet or vault with
any other party; "  
WTF!  
This guy is trying to pull some serious sleeze!  
Ultimately, the damage has already been done.  The coin and its reputation have been thoroughly sabotaged by its incompetent CEO, and the community of token holders that once believed in and supported the coin have been raped by this contrived fiasco.    Many thousands of dollars have been lost by token holders being forced to sell at a loss  as the exchange announced it was to remove the coin.    In my opinion Ryanzo should be held responsible to refund the financial damage he has caused through dishonest misrepresentation of the contract as it relates to trading its tokens on an exchange.

**Comment by noelgit** [comment]
@ryanozo if you are so concerned about the regulatory environment, why don't you just buy back all gmt at ico price. Everyone is now calling your ico a money grab. After getting the money, you left investors holding a bag of worthless gmt tokens. It is just a matter of time before this whole thing escalates and reaches mainstream media. Then you will really get in trouble with regulators.

**Comment by audiowhitepaper** [comment]
@ryanozo @freeatnet on the token sale it states the following: https://www.mercuryprotocol.com/files/Radical_App_International_Terms_of_Token_Sale.pdf

3. Scope of Terms
Unless otherwise stated herein, these Terms govern only your purchase of GMT from us during
the Token Sale. Company will have no responsibility for the use of GMT after GMT are sold. 

My question is why are you @ryanozo trying to to take responsibility of the GMT sold in the secondary market by delisting it when it is explicitly state on the token sale terms that the company will have no responsibility for the use of GMT after GMT are sold? 

Furthermore, it explicitly states that unless otherwise stated herein, these Terms govern the purchase of GMT from company during the Token Sale. 

Wherefore, these terms are not applicable to those who purchased such token on etherdelta and or forkdelta. 

@ryanozo you have no legal right to delist our purchased token from forkdelta. 

@freeatnet we humbly request you do not delist GMT from forkdelta.

**Comment by audiowhitepaper** [comment]
@ryanozo even if it is delisted from forkdelta how will you stop us from trading such token on a P2P exchange such as Air Swap.  https://airswap.io/

**Comment by Rangwenzo** [comment]
@ryanozo 

People are asking valid questions.  Particularly interesting to me is why you let this go on for six months then suddenly cared to stop it.  You continue to say "we can't comment on exchanges" and then say that the TOS say exchanges were never allowed.  You could have said that immediately, and again every time someone asked.  Why did you wait until now?  

I can come up with two possible reasons this has happened ... one must be true:

1) The people behind GMT are selfish jerks.  They could have actually told everyone that exchanges weren't allowed, period.  Allowing it to continue allowed for more hype, more value, and more investors.  So they allowed it to continue to serve their own selfish needs.  Huge, jerk move.

2) The people behind GMT are covering their backends in the face of regulation.  They clearly have been striving to comply the whole time.  They have advisers helping with that.  I can imagine they had another round of regulation compliance checks come through in April and someone said "looks good, although your coin is on exchanges and it'd look good if we had a statement we could show the SEC that we tried to stop that."  So they make a statement.  

Ultimately, @ryanozo simply can't respond to this information.  If it's #1, he's admitting that they've mistreated everyone terribly.  Will someone capable of acting like that admit to it?  No.  If it's #2, admitting to that will invalidate the whole purpose of the statement.  Telling us they made the statement to cover their butts with the SEC will make the SEC ignore the statement.  

So we're never going to get answers.  

In terms of this coin staying on FD: it has to remain.  As people have excerpted from the TOS, the wording is not sufficient to demand the removal.  Taking the coin off would open the door for people to misquote and misinterpret TOS from any/all other coins and make demands for their removals.
